Tests/SonarQube.MSBuild.Tasks.IntegrationTests/TargetsTests/WriteProjectInfoFileTargetTests.cs

```diff
@@ -427,9 +427,7 @@ public void WriteProjectInfo_AnalysisSettings()
             AssertSettingExists(projectInfo, "valid.setting2...", "value 2 with spaces");
             AssertSettingExists(projectInfo, "valid.path", @"d:\aaa\bbb.txt");
             AssertSettingExists(projectInfo, "common.setting.name", "local value");
-
-            Assert.AreEqual(4, projectInfo.AnalysisSettings.Count, "Unexpected number of analysis settings returned");
-
+            // Additional settings might be added by other targets so we won't check the total number of settings
```

Comment:
The change to the targets made this test fail. The assumption in this test that nothing else will be contributing settings is no longer valid so I removed the check.